tests/automation/utilities/reply_message.ts

```diff
@@ -6,13 +6,26 @@ import {
   waitForTextMessage,
 } from './utils';
 
+/**
+ * Reply to a message and optionally wait for the reply to be received.
+ * @param windowSender send the message from this window
```

Comment:
windowSender as a name is confusing,  i'd rather stick to user A and user B (as userA is always the sender)

Reply:
made a change for that, to make the replyTo arguments explicit